modules/ROOT/pages/branded-emails.adoc

```diff
@@ -0,0 +1,66 @@
+= Optionally Configure Branded Emails
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager sends notification emails when an API contract
+is approved or revoked. By default these emails use Anypoint addresses and
+branding. If you wish to configure them to use your own addresses and branding
+instead, use these steps.
+
+. Navigate to *Setup* > *Home* > *Integrations* > *External Data Sources* and ensure *Eligible for External Data Change Capture* is enabled.
+. Set *Polling Interval for External Change Data Capture (seconds)*.
++
+The default is 1800 and the minimum is 300.
+. Navigate to *Setup*, use *Quick Find* to go to *Change Data Capture*, navigate to *Entities* > *Change Manually* > *Available Entities*, and move *AnypointContracts* to the *Selected Entities* list.
+. Navigate to *Setup*, use *Quick Find* to go to *Apex Triggers*, and click the *Developer Console* button.
+. Click *File* > *New* > *Apex Trigger*, set *Name* to `AnypointEmailNotificationsTrigger`, and set *sObject* to `acm_pkg__AnypointContracts__ChangeEvent`.
+. Enter this code block:
++
+----
+    trigger AnypointEmailNotificationsTrigger on acm_pkg__AnypointContracts__ChangeEvent (after insert) {
+        List<acm_pkg__AnypointContracts__ChangeEvent> events = Trigger.new;
+        acm_pkg__.AnypointEmailNotifications.notifyOnContractUpdates(events);
+    }
+----
++
+. Grant the *Organization Administrator* role to the Anypoint user named *Automated Process*.
++
+The impersonation configuration you performed created this Anypoint user automatically. It works with the Salesforce user named *Automated Process* to process triggers. The Anypoint user needs the *Organization Administrator* role to process the API contract events when API contracts are approved or revoked.
+. Navigate to *Setup* and use *Quick Find* to go to *Organization-Wide Addresses*.
+. Set *Display Name* to `EventEmailSender`.
+. Set *Email Address* to the email address at your organization that will send the API contract event notification emails.
```

Comment:
Finding a way around future tense is ideal, but I cannot think of one in this case.

Reply:
Good point. If OK with you, I'll use "the email address at your organization that sends the API contract event notification emails."